### PR TITLE
[Bug fix] Fix skipped tests 

### LINKED ISSUE #13: stuart_ci_build skipped plugin/tests don't show up as skipped
when plugin is skipped it is not recorded as skipped.  No indication that it even exists.  This is not desired behavior.

### PR DESCRIPTION
**Pull request by matthewfcarlson**
Previously, when a test is meant to be skipped by the CI build it will not load the plugin, which means it doesn't show up in the JUnit XML. This fixes that. Fixes #13

### CODE REVIEW

File: edk2toolext/environment/plugin_manager.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import os
 import imp
 import logging
-from edk2toolext.environment import shell_environment
```
Comment (corthon) [inline edk2toolext/environment/plugin_manager.py]: By moving this to ci_build, is the implication that build_plugins can no longer be skipped on a per-platform basis?

File: edk2toolext/invocables/edk2_ci_build.py [hunk 2]
```diff
@@ -252,8 +253,17 @@ def Go(self):
                     # perhaps we should ask the validator to run on the package for this target
 
                     # Still need to see if the package decided this should be skipped
-                    if pkg_plugin_configuration is None or\
-                            "skip" in pkg_plugin_configuration and pkg_plugin_configuration["skip"]:
+                    should_skip = False
+                    if pkg_plugin_configuration is None:
+                        should_skip = True
```
Comment (corthon) [inline edk2toolext/invocables/edk2_ci_build.py]: By not supplying a per-package configuration, the package is skipping all plugins?

**Comment by matthewfcarlson** [comment]
Fixes #13

**Comment by spbrogan** [comment]
closing out.  There are numerous ways to skip a plugin and some will report and others will not.  User can get their desired behavior.

**Pull request closed without merging**